Utilities/DBHelper.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Fidobot.Models;
+using Fidobot.Models.DB;
+using LiteDB;
+
+namespace Fidobot.Utilities;
+
+public static class DBHelper {
+  private static readonly LiteDatabase db = new(@"./main.db");
```

Comment:
I'm going to need this to be configurable. Many online cloud providers will treat the folder with the exes/code in as ephemeral. Containers for example should contain the app inside of them, but we want the data to be stored outside of the containers virtual drive (since upgrade / reinstalling a container will create a new drive).

Reply:
Fixed with f6ce483dca2bf3ec24d3de52bebbde62ee77d215